src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-types/lancie-admin-ticket-types.html

```diff
@@ -0,0 +1,258 @@
+          <paper-input label="Price" name="price" step=".01" type="number" auto-validate required error-message="This field is required!"></paper-input>
+          <paper-input label="Number Available" name="numberAvailable" type="number" auto-validate required error-message="This field is required!"></paper-input>
+          <paper-input label="Deadline" name="deadline" type="datetime-local" auto-validate required error-message="This field is required!"></paper-input>
+          <paper-dropdown-menu label="Buyable" name="buyable" auto-validate required error-message="This field is required!">
+            <paper-listbox slot="dropdown-content">
+              <paper-item>true</paper-item>
+              <paper-item>false</paper-item>
+            </paper-listbox>
+          </paper-dropdown-menu>
+        </lancie-form>
+      </div>
+      <div class="dialog-actions">
+        <paper-button on-tap="tryAddType">Add</paper-button>
+      </div>
+    </lancie-dialog>
+
+    <lancie-dialog id="addOptionToTypeDialog">
+      <h2>Add Options to Type</h2>
+      <div class="dialog-content">
+        <h4>Options will be added to the following Ticket Types</h4>
+        <template id="typeSelected" is="dom-repeat" items="[[selectedTypes]]">
+          <p>[[item.name]]</p>
+        </template>
+        <h4>What Option do you want to add?</h4>
+        <paper-dropdown-menu label="Ticket Option">
+          <paper-listbox slot="dropdown-content" on-iron-select="optionSelected">
+            <template is="dom-repeat" items="[[options]]">
+              <paper-item>[[item.name]]</paper-item>
+            </template>
+          </paper-listbox>
+        </paper-dropdown-menu>
+      </div>
+
+      <div class="dialog-actions">
+        <paper-button dialog-dismiss>Cancel</paper-button>
+        <paper-button on-tap="tryAddOptionToTypes">Add Option</paper-button>
+      </div>
+    </lancie-dialog>
+
+    <lancie-dialog id="changeBuyableOfTypeDialog">
+      <h2>Change Buyable Of Type</h2>
+      <div class="dialog-content">
+        <h4>Buyable will be changed for the following Types:</h4>
+        <template id="typeSelected" is="dom-repeat" items="[[selectedTypes]]">
+          <p>[[item.name]]</p>
+        </template>
+      </div>
+
+      <div class="dialog-actions">
+        <paper-button dialog-dismiss>Cancel</paper-button>
+        <paper-button on-tap="tryChangeBuyableOfTypes">Confirm</paper-button>
+      </div>
+    </lancie-dialog>
+
+  </template>
+  <script>
+    (function () {
+      'use strict';
+
+      Polymer({
+        is: 'lancie-admin-ticket-types',
+        properties: {
+          options: Object,
+          data: Object,
+          mappings: {
+            type: Object,
+            value: function() {
+              return {
+              };
+            }
+          },
+          renderers: {
+            type: Object,
+            value: function() {
+              return {
+                possibleOptions: function(data) {
```

Comment:
You can make this a lambda instead of an anonymous function: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions